### PR TITLE
Toggle menu

### PR DESCRIPTION
**Pull request by Camilovelag**
Good morning/afternoon. 🌤️

This pull request is intended to check the toggle menu in the mobile-version portfolio.

Changes made: 🔧

✔️ Close menu icon was added.
✔️ scripts.js was created and linked to index.html.
✔️ Menu layout was set up for the mobile version.
✔️ Toggle menu was set.

I am expected for feedback.

Regards! 😸

### CODE REVIEW

File: scripts.js [hunk 1]
```diff
@@ -0,0 +1,10 @@
+function showMenu() {
+  let div = document.querySelector('#links-container');
+  div.classList.toggle('active-menu');
+
+  let div2 = document.querySelector('#hamburger-btn');
+  div2.classList.toggle('switch-btn');
+
+  let div3 = document.querySelector('#header-logo');
+  div3.classList.toggle('active-logo');
+  }
```
Comment (Bondok6) [inline scripts.js]: - [x] According to the requirements when the user clicks (or taps) any of the mobile menu options (Links), the mobile menu disappears. so kindly implement it 👌

image

File: index.html [hunk 2]
```diff
@@ -9,17 +9,21 @@
...
-          <a href="#" class="header-link">About</a>
-          <a href="#" class="header-link">Contact</a>
+          <div class="links-container active-menu" id="links-container">
+            <a href="#recent-works" class="header-link">Portfolio</a>
+            <a href="#about-me" class="header-link">About</a>
+            <a href="#contact-us-container" class="header-link">Contact</a>
+          </div>
           <img class="mail-icon" src="images/icon_mail.png" alt="mail_icon">
+          <button class="hamburger-btn switch-btn" id="hamburger-btn" onclick="showMenu()">
+          </button>
```
Comment (Bondok6) [inline index.html]: - [ ] Great job implementing the functionality of `nav-menu` 👌💯, but On the `desktop version` the hamburger menu button and the exit icon **should not be visible**. the nav-menu should work only on the `mobile version` 👌
- [ ] The `desktop-version` should look like your Figma design ( should have all the links in a row ), so kindly fix it 👌 

| Your Design  | Figma Design  |   
| --- | --- |
|   |  |

**Review by Bondok6 — CHANGES_REQUESTED** [review]
## Hi @sidiksam, @Flunavgt 🙋‍♂️, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!



## Highlights

- Great using Gitflow ✅
- Well file structure ✅
- Good summary description ✅

## Required Changes ♻️
- [x] I noticed you didn't add the javaScript Linter, and according to the requirements you should add the [JavaScript + HTML + CSS linter](https://github.com/microverseinc/linters-config/tree/master/html-css-js), so Kindly add it 👌

image


- [x] _Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 2 above)
Comment (Camilovelag) [reply index.html]: image

**Review by cynthiainga — APPROVED** [review]
## STATUS APPROVED :heavy_check_mark: :heavy_check_mark: :heavy_check_mark: 

Hi @Camilovelag, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights

- No linters errors :heavy_check_mark: 
- Good descriptive pull request :heavy_check_mark: 
- Good commit history :heavy_check_mark: 
- Good implementation of required changes :heavy_check_mark: 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**